### ISSUE TITLE
Not all variable font axes work.

### ISSUE THREAD

**Issue by thomgb** [typemytype/drawbot#165]
I have a font with 3 axes. One interpolates (wght) two don't (STEP, ANGL) = featureVariations.
With `fontVariations()` wght and ANGL work fine, but STEP doesn't. STEP sometimes(?) behaves like ANGL...

In FontView all works fine.

**Comment by typemytype** [comment]
could you provide an example font?

**Comment by thomgb** [comment]
Sure.
[PanType.zip](https://github.com/typemytype/drawbot/files/1603501/PanType.zip)

**Comment by justvanrossum** [comment]
Can you verify the font works as expected in a browser, say, with Axis-Praxis? We can't rule out that there are bugs in the OS support for FeatureVariations. I'm not sure which browsers use which parts of OS support for variable fonts, though.

**Comment by typemytype** [comment]
and what OS are you using?

variations works half and half on 10.10

**Comment by thomgb** [comment]
i'm on 10.13

In AP all works good. :)
<img width="1354" alt="screen shot 2018-01-04 at 15 37 57" src="https://user-images.githubusercontent.com/1517937/34568186-4df1b4d0-f165-11e7-887c-1bfce0738415.png">

**Comment by justvanrossum** [comment]
Ha, it works even in Chrome on 10.10 :) So apparently Chrome doesn't use (much of) CoreText.

**Comment by justvanrossum** [comment]
I sent Thom a script that works directly with CoreText and that seems to display the same problem. So: either we're not using CoreText correctly (my script roughly does the same as DB), or CoreText has a bug with FeatureVariations. I optimistically guess it's the latter :)

**Comment by thomgb** [comment]
The same Axis-Praxis test in Safari failed! Must be a CoreText bug.

**Comment by justvanrossum** [comment]
Yes, everything now points to this being a CoreText bug. @thomgb, I assume you are on the latest version of 10.13? @typemytype, do you know who to contact at Apple about this?

**Comment by thomgb** [comment]
@justvanrossum indeed 10.13

**Comment by litherum** [comment]
You can file a bug at bugreporter.apple.com. If you do that, please post the bug number so I can make sure it doesn't fall through the cracks.

**Comment by justvanrossum** [comment]
I think @typemytype filed an issue, and it seems Paolo found it. Frederik, do you have an issue number to share here for reference?

Let's leave this issue open until there's a confirmed fix from Apple.

**Comment by typemytype** [comment]
got notified that is has been resolved in the latest macOS 10.13.4beta, I dont have time to install it...
